### DISCUSSION TITLE
How to display list index in custom track row

### DISCUSSION THREAD

**Discussion by zwr-123** [TT-ReBORN/Georgia-ReBORN#164] [Q&A] [answered]
my setting:     %indexnumber% - %title% - %artist%
display      :       ? - The Scientist - Coldplay

**Comment by TT-ReBORN** [comment]
Hi @zwr-123,

try top menu Options > Playlist > Track row > Show index numbers

-TT

**Reply by zwr-123** [reply]
i tried , then display : artist - album - index.title ,but i want to display  index - title  one row

**Reply by TT-ReBORN** [reply]
Does work as expected:
Top menu `Options `> `Playlist` > `Track row` > `Customize track row`
In both input fields paste: `%title% - %artist%`
Then go to `Options` > `Playlist` > `Track row` > `Show index numbers`
Result: `index number.` `title` - `artist`

**Reply by zwr-123** [reply]
still only display :  title - artist

**Reply by zwr-123** [reply]
When I enable the Album header, it works , but i don't want to see  Album header , I want a playlist like this 
index - title0 - artist0
index - title1 - artist1
index - title2 - artist2

**Reply by TT-ReBORN** [reply, marked as answer]
Ok, you are right. This was not possible when playlist album header was deactivated.
But I have added support in the latest commit https://github.com/TT-ReBORN/Georgia-ReBORN/commit/0ee62db13908d168c6eee94d96234b72652ba47c. 

Download the latest master version as of now from here:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip
and replace the files. 
Now it should work also when the playlist album header is deactivated.

-TT

**Reply by zwr-123** [reply]
thanks,love you
